monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ComponentControllerV1.java

```diff
@@ -474,4 +501,30 @@ static void setPropertyType(ComponentDescriptor<?> descriptor, ConfiguredPropert
             propInfo.setSchema(visitor.finalSchema());
         }
     }
+
+    public enum OutputStyle {
+
+        TABULAR,
+        MAP;
+
+        public static final String HTTP_PARAM_TABULAR = "tabular";
+        public static final String HTTP_PARAM_MAP = "map";
+        public static final String HTTP_PARAM_DOCUMENT = "document";
+
+        public static OutputStyle forString(String outputStyle) {
+
+            if(outputStyle == null) {
```

Comment:
Maybe also check for empty string. If the client is a tool it might easily submit an empty HTTP parameter instead of not including the param.